packages/@biomejs/wasm-web/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@biomejs/wasm-web",
+  "name": "@biomejs/biome_wasm",
```

Comment:
Can you revert this?

Reply:
Weird, I didn't change this file explicitly.